File: nvxapp.server.service/ClientServer-Service/GestionePresenze/My_template1Service/Models/My_Template1Model.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace nvxapp.server.service.ClientServer_Service.GestionePresenze.My_template1Service.Models
{
    public class My_Template1Model
    {
        public int Id { get; set; }
        public int IdDip_RapportoLavoro { get; set; }
    }

    // Per GetAll
    public class My_template1InModel
    {
    }

    public class My_template1OutModel : ModelResult
    {
        public List<My_Template1Model> my_Template1 { get; set; }
        public My_template1OutModel()
        {
            my_Template1 = new List<My_Template1Model>();
        }
    }

    // Per Get (singolo)
    public class My_template1_GetInModel
    {
        public int Id { get; set; }
    }

    public class My_template1_GetOutModel : ModelResult
    {
        public My_Template1Model my_Template1 { get; set; } = new My_Template1Model();
    }

    // Per Put (salvataggio)
    public class My_template1_PutInModel
    {
        public My_Template1Model my_Template1 { get; set; } = new My_Template1Model();
    }

    public class My_template1_PutOutModel : ModelResult
    {
        public My_Template1Model my_Template1 { get; set; } = new My_Template1Model();
    }
}

File: nvxapp.client/src/nvx/ClientServer-Service/GestionePresenze/My_Template1Service/Models/my-template1-model.ts
import { ModelResult } from "../../../ModelsBase/model-result";

export class My_Template1Model {

  public id: number;
  public idDip_RapportoLavoro!: number; 
    

}

export class My_template1_GetAllInModel {
  
}

export class My_template1_GetAllOutModel extends ModelResult {

  public my_Template1: My_Template1Model[];

}


export class My_template1_GetInModel {
  public id: number;
}
export class My_template1_GetOutModel extends ModelResult {
  public my_Template1: My_Template1Model;
}
export class My_template1_PutInModel {
  public my_Template1: My_Template1Model;
}
export class My_template1_PutOutModel extends ModelResult {
  public my_Template1: My_Template1Model;
}


